randovania/games/samus_returns/logic_database/Area 5 Tower Interior.txt

```diff
@@ -449,31 +485,11 @@ Extra - asset_id: collision_camera_007
               # Screw only: https://youtu.be/eI3DtHqx5dU?t=43
               Movement (Expert)
 
-> Event - Grapple Block; Heals? False
-  * Layers: default
-  * Event Area 5 (Tower Interior) - Grapple Shuffler Grapple Block
-  > Above Pickup
-      Trivial
-
-> Above Pickup; Heals? False
+> Event - Shuffler Grapple Block; Heals? False
```

Comment:
can you rename this node to "shuffler long grapple block" for clarity purposes please?

Reply:
Fixed